Interop/Runtime/src/native/kotlin/kotlinx/cinterop/NativeUtils.kt

```diff
@@ -51,6 +51,9 @@ external fun <R : Number> Number.narrow(): R
 @Retention(AnnotationRetention.SOURCE)
 internal annotation class JvmName(val name: String)
 
+fun cValuesOf(vararg elements: Byte): CValues<ByteVar> =
+        createValues(elements.size) { index -> this.value = elements[index] }
```

Comment:
What's the motivation behind replacing original implementation but this one? It seems to be less efficient.